lib/logdna.rb

```diff
@@ -20,50 +20,24 @@ def initialize(key, opts={})
       @level = opts[:level] || 'INFO'
       @env = opts[:env]
       @meta = opts[:meta]
-      @@client = nil unless defined? @@client
```

Comment:
If you're going to remove this we might as well make this `@client` (instance var) instead of `@@client` (class var).